.github/workflows/push_docker_test_container.yaml

```diff
@@ -0,0 +1,35 @@
+name: Publish Docker Test Container
+
+on:
+  push:
+    branches:
+      - master
+      - main
+      - 'ci-release-test/**'
+
+jobs:
+  publish-test-container:
+    runs-on: ubuntu-latest
+    environment: publish
+    steps:
+      - uses: actions/checkout@v2
+      - name: Init submodules
+        run: git submodule update --init --recursive
+      - name: Build TestContainer
+        run: ./exaslct build-test-container --force-rebuild
+      - name: Push TestContainer image
+        if: ${{ (github.ref == 'refs/heads/master') || (github.ref == 'refs/heads/main') }}
+        run: |
+          ./exaslct push-test-container --source-docker-username "$SECRET_DOCKER_USER_NAME" --target-docker-username "$SECRET_DOCKER_USER_NAME" --force-push
```

Comment:
Same user used for source and target?
Also the passwords are not used.

Reply:
exaslct uses the password environment variables internally, such that we don't need to include them into the CLI. And, yes source user and target user are here the same. Because, source and target docker repository are the same.